src/components/dateTimePicker/DateTimePicker.driver.ts

```diff
@@ -0,0 +1,31 @@
+import {fireEvent} from '@testing-library/react-native';
+import {
+  ComponentProps,
+  ExpandableOverlayDriver,
+  ButtonDriver
+} from '../../testkit';
+
+export const DateTimePickerDriver = (props: ComponentProps) => {
+  const {renderTree, testID} = props;
+  const expandableOverlayDriver = ExpandableOverlayDriver({...props, testID: `${testID}.overlay`});
+  const openPicker = () => {
+    expandableOverlayDriver.open();
+  };
+  const discardPicker = () => {
+    ButtonDriver({...props, testID: `${testID}.cancel`}).press();
+  };
+  const submitSelection = () => {
+    ButtonDriver({...props, testID: `${testID}.done`}).press();
+  };
+  const changeDateTo = (date: Date) => {
```

Comment:
Maybe `selectDate`?

Reply:
But it doesn't really `select` the value. It only invokes the onChange of the picker. The user has to submit this date (by clicking the done button).